src/index.js

```diff
@@ -2,22 +2,83 @@ import 'lodash';
 import './style.css';
 import './queries.css';
 
-const addScore = () => {
-  let name = document.querySelector('.name').value;
-  let score = document.querySelector('.score').value;
+async function getData(url) {
+  const myRequest = new Request(url, {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    mode: 'cors',
+    cache: 'default',
+  });
+  const data = await fetch(myRequest).then(
+    (response) => response.json(),
+  ).then((json) => json.result);
+  return data;
+}
+function addScore() {
+  const name = document.querySelector('.name').value;
+  const score = Number(document.querySelector('.score').value);
```

Comment:
- [ ] Consider using arrow function instead of function keyword as arrow functions allow us to write shorter function syntax and ensure we're following ES6 standard. 😉